deploy/osd-namespace-labels/README.md

```diff
@@ -0,0 +1,13 @@
+# Overfiew
```

Comment:
Typo in “overview”

Reply:
https://github.com/openshift/managed-cluster-config/pull/375